package.json

```diff
@@ -52,6 +54,7 @@
     "lint-staged": "^10.5.3",
     "prettier": "^2.0.5",
     "snowpack": "^3.0.1",
+    "ts-jest": "^26.5.0",
```

Comment:
This doesn't appear to be used anywhere... Do we need it?

Reply:
i am not sure. shall i remove it?